src/app/public/modules/field-masker/field-masker-input.directive.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  Directive,
+  ElementRef,
+  HostListener,
+  Input,
+  OnInit,
+  Optional
+} from '@angular/core';
+
+import 'rxjs/add/operator/distinctUntilChanged';
+
+import 'rxjs/add/operator/takeUntil';
+
+import {
+  SkyDatepickerComponent,
+  SkyDatepickerConfigService
+} from '../datepicker';
+
+import {
+  GroupLogicService
+} from './group-logic.service';
+
+@Directive({
+  selector: '[skyFieldMaskerInput]'
+})
+export class SkyFieldMaskerInputDirective implements OnInit {
```

Comment:
the class and selector i'm fine with changing but the `dateFormat` input shouldn't change because we want it to be the same as the datepicker input directive's format